.devcontainer/Dockerfile

```diff
@@ -35,15 +37,22 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 RUN update-ca-certificates
 
 # Terraform
-RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - \
-	&& apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
-	&& apt-get update && sudo apt-get -qq install --no-install-recommends --yes terraform terraform-ls
+# hashicorp only provides amd64 packages, so we have to make do
+ARG TERRAFORM_VERSION=1.3.6
+RUN mkdir -p /tmp/download \
+	&& cd /tmp/download \
+	&& wget -O terraform.zip "https://releases.hashicorp.com/terraform/1.3.6/terraform_1.3.6_linux_${TARGETARCH}.zip" \
```

Comment:
Hi @DavidS-om , thank you for your PR.
The `TERRAFORM_VERSION` is not being used here. Would be great to allow the user to pass it on as arguments.

Reply:
d'oh, yes!